sbol3/document.py

```diff
@@ -424,22 +424,25 @@ def join_lines(self, lines: List[Union[bytes, str]]) -> Union[bytes, str]:
         lines_type = type(lines[0])
         if lines_type is bytes:
             # rdflib 5
-            return b''.join(lines)
+            return b'\n'.join(lines) + b'\n'
```

Comment:
Will this cause any issue for Windows expecting its "two-character newline", or is string-writing smart about that?